### DISCUSSION TITLE
is facets supported in zinc?

### DISCUSSION THREAD

**Discussion by mjerjawi** [zinclabs/zinc#122] [Q&A]
Hello to every one i really like this amazing project 
and i would like to know if Facets search supported in zinc like ElasticSearch Facets 
https://www.elastic.co/guide/en/app-search/current/facets-guide.html

**Comment by prabhatsharma** [comment]
You can use aggregations to achieve a similar result in Zinc. https://docs.zinclabs.io/API%20Reference/search/aggregation/
